src/processes.c

```diff
@@ -230,7 +234,7 @@ reuse_or_add_context (struct ipc_context *ctx)
  * FORKFAILED
  */
 pid_t
-create_ipc_process (ipc_process_func func, void *args)
+create_ipc_process (kb_t main_kb, ipc_process_func func, void *args)
```

Comment:
If main kb is set via `set_main_kb()` in the main process (in openvas.c) at the beginning of the scan and stored it in global variable, it shouldn't be necessary to pass it as argument, since it is accessible from everywhere and from every forked process.